src/WebJobs.Script.Host/Program.cs

```diff
@@ -19,11 +23,38 @@ static void Main(string[] args)
 
             ScriptHostConfiguration config = new ScriptHostConfiguration()
             {
-                RootScriptPath = rootPath
+                RootScriptPath = rootPath,
+                FileLoggingEnabled = true
```

Comment:
Why adding this? The intent for this host is that it is for general continuous WebJobs use, where Dashboard logging is used, not file logging.